bin/install-sdk

```diff
@@ -17,8 +17,14 @@ else
 	urls=$(curl -s $SDK_FETCH_URL | grep download_url | sed 's/.*: "\(.*\)"/\1/')
 fi
 if [[ -z "$urls" ]]; then
-	echo "ERROR: api.github.com request failed?!" >&2
-	exit 1
+	echo "GitHub API request failed. Please set the GITHUB_TOKEN environment variable and try again."
+        echo "Dropping you to a shell, type exit when done."
+        bash
```

Comment:
variables from the child shell won't impact the calling process.
I think just adding the line above (the `echo` to tell the user about this solution) is sufficient.